src/net/mqtt_interface.rs

```diff
@@ -0,0 +1,235 @@
+use crate::hardware::{
+    design_parameters::MQTT_BROKER, CycleCounter, EthernetPhy, NetworkStack,
+};
+
+use core::{cell::RefCell, fmt::Write};
+
+use heapless::{consts, String};
+use serde::Serialize;
+
+use super::{Action, MqttMessage, SettingsResponse, SettingsResponseCode};
+
+/// MQTT settings interface.
+pub struct MqttInterface<S>
```

Comment:
I think we should work towards making this stabilizer-agnostic as well. We'd want the same in Thermostat and Booster.

Reply:
As per discussion, this will move to `miniconf` in the future as a generic API behind an mqtt feature-flag.